src/types/store/describe.ts

```diff
@@ -18,9 +18,15 @@ export interface IDescribeState {
     error?: IResponseError;
 }
 
+export interface IDescribeHandledlResponse {
+    path: string | undefined;
+    data: IDescribeData | undefined;
+    currentDescribe: TEvDescribeSchemeResult | IDescribeData | undefined;
```

Comment:
complex types are always worse than simple ones
this can be always represented as `IDescribeData`, no need to convert to a plain value for the edge case of a single item